commons/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="fr.free.nrw.commons"
+    package="free.nrw.commons"
```

Comment:
I think the reason free.nrw.commons works and fr.free.nrw.commons doesn't is that the former has 3 'tiers' in the folder hierarchy, similar to org.wikimedia.commons, and the latter has 4. In order to get the latter project structure to be valid in the IntelliJ IDEA, I had to create a new folder 'fr' and move the 'free' folder into it. Whereas with the former, I only had to rename the folders. I could not figure out any way to add one tier in the middle of the folder hierarchy without doing the move. That was why initially I was hoping that you would accept free.nrw.commons... :) But then I realized that the package name probably must stay the same if the app has already been published.

What older name do you mean... the package name? But we are only changing the folders, yes, so the package names in the code would not have changed from the initial clone? Unfortunately I develop on a Windows desktop.